app/controllers/concerns/lol_connections.rb

```diff
@@ -1,127 +0,0 @@
-require 'lol'
```

Comment:
Yeah, I will revert the friendship stuff in this pull request so only working code is moved to master.  Once I make that commit here, I will also log a new issue to handle that refactor.